react-app/src/modules/account/account.js

```diff
@@ -0,0 +1,100 @@
+import scrypt from 'scrypt-async';
+import nacl from 'tweetnacl';
+import sha256 from 'fast-sha256';
+import blake from 'blakejs';
+
+nacl.util = require('tweetnacl-util');
+
+let _username, _pk, _sk, _tz, _offset;
+
+export function clearAccount() {
+  _username = _pk = _sk = _tz = _offset = undefined;
+}
+
+export function setKeyPair(pk, sk) {
```

Comment:
In fact, I didn't do in that way because I don't want to export the module properties, in order to be only accessibles by getters/setters. In particular, _pk and _sk must be computed by the constructor.

setKeyPair(_pk, _sk) exists only for testing purposes. I will rewrite it to _setKeyPair(_pk, _sk) (I forgot the _), but the idea is to exclude this piece of code from production.

Reply:
_setKeyPair is a reasonable compromise.

It's unfortunate that you need to export everything for the test.